### PR TITLE
Create Post

### PR DESCRIPTION
**Pull request by juampick**
- Added moment library for dates, mockBlog api now generates the publishedDate. 
- Created ManagePostPage container component
- PostForm component for handling creation. 
- Created actions, actionsTypes 
- Reducer cases to support Post creation
- Routing

(Some things are now prepared for starting the handle on update view)

### CODE REVIEW

File: src/actions/postActions.js [hunk 1]
```diff
@@ -16,6 +24,16 @@ export function loadPosts(){
     }
 }
 
+export function createPost(post){
+    return (dispatch, getState) => {
+        return blogAPI.savePost(post).then(postSaved => {
+            post.id ? dispatch(updatePostSuccess(postSaved)) :
+                dispatch(createPostSuccess(postSaved));
+        }).catch(error => {
```
Comment (mravinale) [inline src/actions/postActions.js]: create and update should be using diferent endpoints

File: src/components/blog/BlogPage.js [hunk 2]
```diff
@@ -15,9 +16,9 @@ class BlogPage extends React.Component {
             <div className="row">
                 <div className="col-md-6 col-md-offset-3">
                     <h2>My Posts</h2>
-                    <a href="#" className="btn btn-primary">
+                    <Link to="post" className="btn btn-primary" activeClassName="active">
```
Comment (mravinale) [inline src/components/blog/BlogPage.js]: 👍

File: src/actions/postActions.js (hunk 1 above)
Comment (juampick) [reply src/actions/postActions.js]: Ok perfect, will separate both now ;)

File: src/components/blog/PostForm.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+PostForm.propTypes = {
+    onChange: PropTypes.func.isRequired,
+    onSave: PropTypes.func.isRequired,
+    saving: PropTypes.bool.isRequired,
+    errors: PropTypes.object
+};
+
+export default PostForm;
```
Comment (mravinale) [inline src/components/blog/PostForm.js]: what do you think about using this plugin for forms? https://github.com/christianalfoni/formsy-react

**Review by mravinale — CHANGES_REQUESTED** [review]

File: src/components/blog/PostForm.js (hunk 3 above)
Comment (juampick) [reply src/components/blog/PostForm.js]: Yes, I saw it, but for first sprint I thought to finish the complete functionality on traditional way, and after that improve it. But If you like I can do it right now.

File: src/actions/postActions.js [hunk 4]
```diff
@@ -20,24 +20,20 @@ export function loadPosts(){
...
-            post.id ? dispatch(updatePostSuccess(postSaved)) :
-                dispatch(createPostSuccess(postSaved));
+        return blogAPI.createPost(post).then(postSaved => {
+            dispatch(createPostSuccess(postSaved));
         }).catch(error => {
             throw(error);
-        })
-    }
+        });
+    };
```
Comment (mravinale) [inline src/actions/postActions.js]: 👍

**Comment by juampick** [comment]
`Formsy` implemented.
Branch rebased with latest _master_ changes. 
Now going to create and call a createPost endpoint.

**Review by mravinale — APPROVED** [review]

**Pull request merged**